main.nf

```diff
@@ -95,10 +131,15 @@ workflow {
     )
 
     //
-    // WORKFLOW: Run main workflow
+    // Filter genome assembly by minimum contig length
+    //
+    SEQKIT_GET_LENGTH(PIPELINE_INITIALISATION.out.target_assembly)
```

Comment:
`SEQKIT_GET_LENGTH` should be part of the `GENEPAL` workflow defined in `workflows/genepal.nf` file. This structure is also inherited from the nf-core template and allows creating of meta-pipelines where two are more pipelines can be joined into a larger single pipeline.